content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-rego-policies.md

```diff
@@ -5,7 +5,7 @@ description: "Writing Rego-based policies for Chainguard Enforce"
 date: 2023-01-12T15:56:52-07:00
 lastmod: 2023-01-12T15:56:52-07:00
 draft: false
-images: []
+images: ["SBOM"]
```

Comment:
I think this should be the new tags field?

```suggestion
tags: ["SBOM"]
```

Reply:
oh gosh, thanks